setup.py

```diff
@@ -12,6 +12,9 @@
     print("warning: pypandoc module not found, could not convert Markdown to RST")
     long_description = ''
 
+with open('requirements.txt') as f:
+    required = f.read().splitlines()
```

Comment:
Yeah, that's the typical thing.  You could have two reqs files and only open one for install.  I just kept it simple here to be illustrative.